src/constants/contracts.ts

```diff
@@ -446,6 +447,18 @@ export const CONTRACT_TOKEN_ADDRESS = isOnTestnet
       },
     };
 
+interface IContractTokenAddress {
```

Comment:
I think that name is confusing. This isn't just the address, but the whole token. Perhaps we can simply name it `Token`.